src/Engine/Manager/PhysicsManager.hpp

```diff
@@ -26,4 +41,6 @@ class PhysicsManager : public Physics::Simulator, public SuperManager {
         void operator=(PhysicsManager const&) = delete;
         
         glm::vec3 gravity = glm::vec3(0.f, -9.82f, 0.f);
+        
+        ComponentContainer<Component::Physics> physicses;
```

Comment:
Rename to physicComponents or similar, anything but physicses really.
Might be subject to change as the phyics library is implemented.